estimagic/tests/conftest.py

```diff
@@ -1,7 +1,58 @@
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
 import pytest
 
 
 @pytest.fixture(scope="function", autouse=True)
 def fresh_directory(tmpdir):
     """Each test is executed in a fresh directory."""
     tmpdir.chdir()
+
+
+@pytest.fixture
```

Comment:
Load once per session? Or do you change it inplace?
```suggestion
@pytest.fixture(scope="session")
```